unittest/test_job.go

```diff
@@ -166,15 +166,12 @@ func (t *TestJob) releaseOption() *chartutil.ReleaseOptions {
 }
 
 // get chartutil.CapabilityOptions ready for render
-// Only supports APIVersions for now
+// Only supports APIVersions, KubeVersion for now
 func (t *TestJob) capabilityOption() *chartutil.Capabilities {
-	options := chartutil.Capabilities{APIVersions: chartutil.DefaultVersionSet}
-	if len(t.Capabilities.APIVersions) > 0 {
-		var arr []string
-		arr = append(t.Capabilities.APIVersions, "v1")
-		options.APIVersions = chartutil.NewVersionSet(arr...)
```

Comment:
Wouldn't this remove the ability for the TestJob to be able to specify a specific `.Capabilities.APIVersions` to test against? Is that no longer required?

Reply:
hey @aiyengar2 - how can i define `.Capabilities.ApiVersions`  for a unit test? I didn't find any examples

Reply:
i got it!

Reply:
I haven't tried it personally, but you should be able to pass in `.Capabilities.ApiVersions` for a particular test via a template file like:
```yaml
suite: test deployment
templates:
  - deployment.yaml
tests:
  - it: should work
    set:
      image.tag: latest
    asserts:
      - isKind:
          of: Deployment
      - matchRegex:
          path: metadata.name
          pattern: -my-chart$
      - equal:
          path: spec.template.spec.containers[0].image
          value: nginx:latest
    capabilities: 
      apiVersions:
      - v1
```

The actual template YAML file is [unmarshalled](https://github.com/rancher/helm-unittest/blob/17719b59b14224a0cc69641e22ca664dede8a63c/unittest/test_suite.go#L16-L36) into the [`TemplateSuite`](https://github.com/rancher/helm-unittest/blob/17719b59b14224a0cc69641e22ca664dede8a63c/unittest/test_suite.go#L39-L48), which contains [`TestJobs`](https://github.com/rancher/helm-unittest/blob/17719b59b14224a0cc69641e22ca664dede8a63c/unittest/test_job.go#L33-L55) that can take in capabilities.